source/code/scxsystemlib/common/GetLinuxOS.sh

```diff
@@ -251,6 +283,7 @@ GetLinuxInfo() {
             if [ "$OSName" = "" ]; then
                 OSName="Linux"
                 OSManufacturer="Universal"
+                OSShortName="$OSName"
```

Comment:
The expected output is if there's no version available, then not to add that in the OSShortName. Thus, no "_".